tests/idl001.html

```diff
@@ -373,10 +373,10 @@ <h2 class="no-num no-ref heading settled" id="idl-index"><span class="content">I
     <li><a href="#ref-for-dom-foo-foo">Unnamed section</a>
    </ul>
   </aside>
-  <aside class="dfn-panel" data-for="dom-foo-_or">
-   <b><a href="#dom-foo-_or">#dom-foo-_or</a></b><b>Referenced in:</b>
+  <aside class="dfn-panel" data-for="dom-foo-or">
+   <b><a href="#dom-foo-or">#dom-foo-or</a></b><b>Referenced in:</b>
    <ul>
-    <li><a href="#ref-for-dom-foo-_or">Unnamed section</a> <a href="#ref-for-dom-foo-_or①">(2)</a>
+    <li><a href="#ref-for-dom-foo-or">Unnamed section</a>
```

Comment:
This change I don't understand and looks like it could be a regression. @tabatkins @plinss?

Reply:
Linking with `{{or}}` still works, but `{{_or}}` doesn't. I think that's fine.

Reply:
Yeah, that does seem just fine.